controllers/managedocs_controller.go

```diff
@@ -108,8 +105,9 @@ const (
 	convergedDeploymentType                = "converged"
 	consumerDeploymentType                 = "consumer"
 	providerDeploymentType                 = "provider"
-	servingCertCABundleConfigMapName       = "prometheus-serving-cert-ca-bundle"
-	prometheusServiceSecretName            = "managed-ocs-prometheus-service-secret"
+	servingCertCABundleConfigMapName       = "prometheus-cert-bundle"
+	prometheusServiceSecretName            = "managed-ocs-prom-svc-secret"
```

Comment:
Why is the value of this constant (secret name) is widely different than the names convention on the surrounding constants

Reply:
Updated the name to `prometheus-svc-secret` in latest commit.